src/checkers/debianrepochecker.py

```diff
@@ -57,10 +63,26 @@
     "var/lib/dpkg/updates",
     "var/lib/dpkg/info",
 )
+DEB_HASH_MAP = {
+    "SHA512": "sha512",
+    "SHA256": "sha256",
+    "SHA1": "sha1",
+    "MD5Sum": "md5",
+}
 
 LOG = logging.getLogger(__name__)
 
 
+def read_deb_hashes(deb_hashes: apt_pkg.HashStringList) -> MultiDigest:
+    digests: t.Dict[str, str] = {}
+    deb_hash: apt_pkg.HashString
+    for deb_hash in deb_hashes:
```

Comment:
Any idea why Mypy complains? `apt_pkg` is an extension module, but Mypy doesn't complain on my machine.